src/template/context.js

```diff
@@ -43,6 +43,31 @@ const context = {
             long: '--.+?(\\s|=|$)',
         },
     },
+    argumentTypes: {
+        string: [
+            'string',
+            'str',
+            'dir',
+            'directory',
+            'path',
+            'file',
+        ],
+        number: [
+            'int',
+            'integer',
+            'number',
+        ],
+        array: [
```

Comment:
We don't have a `array` type

Reply:
I'm not sure, but better put the `argumenTypes` in separate file? Because we use it only for identity types of arguments